src/apps/components/AppAlerts/useAllAppsAlert.ts

```diff
@@ -0,0 +1,53 @@
+import { useUserPermissions } from "@dashboard/auth/hooks/useUserPermissions";
+import { PermissionEnum, useAppFailedPendingWebhooksQuery } from "@dashboard/graphql";
+import { useMemo } from "react";
+
+const requiredPermissions = [PermissionEnum.MANAGE_APPS];
+
+interface FailedWebhooksCount {
+  hasFailed: boolean;
+  hasPending: boolean;
+}
+
+const defaultFailedWebhooksInfo: FailedWebhooksCount = {
+  hasFailed: false,
+  hasPending: false,
+};
+
+export const useAllAppsAlert = (): FailedWebhooksCount => {
+  const permissions = useUserPermissions();
+  const hasRequiredPermissions = requiredPermissions.some(permission =>
+    permissions?.map(e => e.code)?.includes(permission),
+  );
+
+  const { data } = useAppFailedPendingWebhooksQuery({
+    skip: !hasRequiredPermissions,
+  });
+
+  const failedWebhooksInfo = useMemo(
+    () =>
+      data?.apps?.edges.reduce((acc, app) => {
+        const webhookInfo = defaultFailedWebhooksInfo;
+
+        app.node.webhooks?.forEach(webhook => {
+          const { failedDelivers, pendingDelivers } = webhook;
+
+          if (failedDelivers && failedDelivers.edges?.length > 0) {
+            webhookInfo.hasFailed = true;
+          }
+
+          if (pendingDelivers && pendingDelivers.edges?.length > 0) {
+            webhookInfo.hasPending = true;
```

Comment:
You need to fetch attempts as well, here: https://github.com/saleor/saleor-dashboard/pull/5403/files#diff-9f3bbc45c9cc037644ba210310c6e2cb538fa2dfc054166306de7ebd00489e9fR27

webhook that has pending delivers means that they are being sent, not they are failed. You need to show alert ONLY when some of these delivers have FAILED attempt. So here you need to go throughout the attempts array and find some failed elements